### PR TITLE
Bug 1189927 - Content updates need to be made on the FxOS FAQ

### PR DESCRIPTION
**Pull request by malena**
https://bugzilla.mozilla.org/show_bug.cgi?id=1189927

### CODE REVIEW

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 1]
```diff
@@ -21,160 +21,202 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+            <div data-accordion-role="tabpanel">
+                <dl class="faq">
+                    <dt>{{_('How do I…')}}</dt>
+                    <dd>
+                        <ul>
+                            <li><a href="https://support.mozilla.org/kb/make-calls-and-access-voicemail-firefox-os">
+                                {{_('…make a call?')}}
+                            </a></li>
+                            <li><a href="https://support.mozilla.org/kb/import-add-and-manage-contacts-firefox-os">
+                                {{_('…import my contacts? ')}}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/os/faq.html]: Drop the space in the string

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 2]
```diff
@@ -21,160 +21,202 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                            </a></li>
+                            <li><a href="https://support.mozilla.org/kb/how-to-use-messages-app">
+                                {{_('…create and view messages? ')}}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/os/faq.html]: Same here

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 3]
```diff
@@ -21,160 +21,202 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                        To see a full list of available Firefox OS phones, please visit the <a href="{{ link }}">devices page</a>.
+                        {% endtrans %}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('Where can I buy Firefox OS?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        {% trans link=url('firefox.os.devices') %}
+                        The full list of where to buy Firefox OS devices can be found at <a href="{{ link }}">https://www.mozilla.org{{ link }}</a>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/os/faq.html]: Does it make sense to have a text like this for links? Wouldn't it be better to have links with words like "devices page" in this case (see also example above).

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 4]
```diff
@@ -21,160 +21,202 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                    <dd>{{_('Includes directions, traffic and local transit information.')}}</dd>
+                </dl>
+            </div>
+        </section>
+
+        <section>
+            <h2 data-accordion-role="tab">{{_('Privacy and Security')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <dl class="faq">
+                    <dt>{{_('How will Firefox OS keep me safe while I browse?')}}</dt>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/os/faq.html]: This string isn't part of the existing template, should be behind a tag.

**Comment by malena** [comment]
Waiting on one more url confirmation in copy.

**Comment by malena** [comment]
Links confirmed ready for review.

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 5]
```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                        {{_('Many more are being made available every day.')}}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('Is Firefox OS available in my language?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        <p>
+                            {% trans link=url('firefox.os.devices') %}
+                            Firefox OS is currently available in a number of different languages. Check out the device market availability at <a href="{{ link }}">https://www.mozilla.org{{ link }}</a>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/os/faq.html]: This string remained with the URL used as link text? Still wanted?

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 6]
```diff
@@ -11,7 +11,7 @@
 {% block body_id %}firefox-os-faq{% endblock %}
 
 {% block page_css %}
-  {% stylesheet 'firefox_faq' %}
+    {% stylesheet 'firefox_faq' %}
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/os/faq.html]: Nit: I believe indentation in HTML is two spaces?

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 7]
```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                    </dd>
+
+                    <dt>{{_('How is Firefox OS different from other operating systems?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        <p>
+                            <b>{{_('For users')}}</b> {{_('Firefox OS offers a customized and easy to use experience appropriate to consumers from the value range in India to early tech adopters in Japan')}}
+                        </p>
+                        <p>
+                            <b>{{_('For operators and OEMs')}}</b> {{_('Firefox OS offers the opportunity to capture new user segments due to its open development process and tight collaboration with partners searching for new revenue streams and business models ')}}
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/os/faq.html]: Nit: This sentence and the one before is missing a period at the end of the sentence. This one also has an extra leading space at the end.

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 8]
```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                        The full list of where to buy Firefox OS devices can be found at the <a href="{{ link }}">devices page</a>.
+                        {% endtrans %}
+                    {% else %} 
+                        {% trans link=url('firefox.os.devices') %}
+                        To purchase a Firefox OS phone, please visit the <a href="{{ link }}">devices page</a> and select your available country.
+                        {% endtrans %}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('What will be the price of the phone?')}}</dt>
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/os/faq.html]: Should this read: "What will the price of the phone be?"

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 9]
```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                    <dd>{{_('Pricing is not set by Mozilla and will be established by our operator and manufacturer partners based on local market dynamics.')}}</dd>
+
+                    <dt>{{_('What apps will be available on Firefox OS?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        <p>{{_('The Firefox Marketplace offers an open app environment that leverages the scale of the Web as a development platform; allowing operators, phone manufacturers and developers greater flexibility.')}}</p>
+                        <p>
+                            {% trans url='https://marketplace.firefox.com/' %}
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/os/faq.html]: Nit: Maybe use `link` for consistency with the rest of the page?

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 10]
```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+                    {% endtrans %}
+                    </dd>
+                </dl>
+            </div>
+        </section>
+
+        <section>
+            <h2 data-accordion-role="tab">{{_('Developer Questions')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <p>{% trans url='https://developer.mozilla.org/docs/Mozilla/Firefox_OS' %}
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/os/faq.html]: Again here, perhaps use `link`

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 11]
```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
...
+            </div>
+        </section>
+    </div>
 </div>
 
 {% endblock %}
 
 {% block js %}
-  {% javascript 'firefox_faq' %}
+    {% javascript 'firefox_faq' %}
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/os/faq.html]: Two spaces indentation.

File: bedrock/firefox/templates/firefox/os/faq.html (hunk 10 above)
Comment (flodolo) [inline bedrock/firefox/templates/firefox/os/faq.html]: Please note that every change like this, on a string that is not new (i.e. behind a tag), is going to hurt l10n. So, please don't change them ;-)

File: bedrock/firefox/templates/firefox/os/faq.html (hunk 8 above)
Comment (malena) [inline bedrock/firefox/templates/firefox/os/faq.html]: This line is currently live and is not part of the copy updates. Will leave as is.

File: bedrock/firefox/templates/firefox/os/faq.html (hunk 7 above)
Comment (malena) [inline bedrock/firefox/templates/firefox/os/faq.html]: Great catch will update.

File: bedrock/firefox/templates/firefox/os/faq.html (hunk 9 above)
Comment (malena) [inline bedrock/firefox/templates/firefox/os/faq.html]: I believe the reason why there are both "link" and "url" variable names is that, link points to a page in mozilla.org while url points to an external page. Will keep as is for the sake of flod's comments below.

File: bedrock/firefox/templates/firefox/os/faq.html (hunk 11 above)
Comment (malena) [inline bedrock/firefox/templates/firefox/os/faq.html]: Ahh okay I have updated my editor to use 2-spaces as well.

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 12]
```diff
@@ -76,41 +99,60 @@ <h2 data-accordion-role="tab">{{_('Buying Firefox OS')}}</h2>
...
 
           <dt>{{_('Is Firefox OS available in my language?')}}</dt>
           <dd>
-          {{_('Firefox OS is currently available in Spanish, Polish, English, Italian, Portuguese, German, Serbian, Hungarian and Greek.')}}
-          {{_('We’re working on adding more languages all the time.')}}</dd>
-
+          {% if l10n_has_tag('fxos_faq_updates') %}
+            <p>
+              {% trans link=url('firefox.os.devices') %}
+              Firefox OS is currently available in a number of different languages. Check out the device market availability at the <a href="{{ link }}">the device.</a>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/os/faq.html]: I think you meant to use `at the <a href="%(link)s">devices page</a>` here, the current link text is wrong.

**Comment by flodolo** [comment]
@malena There's one small issue with one string. Once that's fixed, do you think we're good to extract strings? I have a local file ready, just need to make sure the last one matches the code.

File: bedrock/firefox/templates/firefox/os/faq.html [hunk 13]
```diff
@@ -118,63 +160,67 @@ <h2 data-accordion-role="tab">{{_('Using Firefox OS')}}</h2>
...
   </div>
-
 </div>
 
 {% endblock %}
 
 {% block js %}
   {% javascript 'firefox_faq' %}
-{% endblock %}
+{% endblock %}
```
Comment (schalkneethling) [inline bedrock/firefox/templates/firefox/os/faq.html]: Nit: Missing empty line at end of file

**Comment by malena** [comment]
@flodolo Okay those last few issues have been addressed. I think we are good now :)

**Comment by flodolo** [comment]
Thanks. I'm waiting for another needinfo on Bugzilla (different page) before pushing out the strings to localizers, but I've already extracted the strings locally, so let's try to keep the strings frozen :+1:

**Comment by schalkneethling** [comment]
@malena @flodolo should we hold of on merging this until some translations exist or, can I merge?

**Comment by flodolo** [comment]
No need to wait for merge (that's why we use tags)

**Comment by schalkneethling** [comment]
Thanks @flodolo - I just wanted to make doubly sure.

**Comment by schalkneethling** [comment]
r+ :cake:

**Pull request merged**